bool is_moving
float64 linear_velocity
float64 angular_velocity
